shesmu-server-ui/src/help.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  UIElement,
+  buttonIcon,
+  subscribedState,
+  addElements,
+  br,
+} from "./html.js";
+import { locallyStored } from "./io.js";
+import { Publisher, pubSubModel } from "./util.js";
+export type Area = "action" | "alert" | "olive" | "pauses" | "simulator";
+/**
+ * Describes an individual help message that should appear.
+ */
+interface HelpTip {
+  /**
+   * A short (one line) description of the feature
+   */
+  summary: string;
+  /**
+   * A long description of the feature
+   */
+  description: string;
+  /**
+   * The hotspot (_i.e._, UI location) where this help tip should be displayed.
+   */
+  hotspot: string | null;
+  /**
+   * A general category for the kind of information in this tip so that everything in a category can be marked as read at once.
+   */
+  area: Area;
+}
+/**
+ * All the help for the whole UI. Each outer array represents a release version of Shesmu.
+ */
+const tips: HelpTip[][] = [
+  [
+    {
+      summary: "Use advanced search for tag×tag filtering",
+      description:
+        "When using the basic search, multiple tags are treated as “any” matches, so selecting another tag will broaden rather than narrow the search. Switch to the advanced search to allow narrowing when searching tags.",
+      hotspot: "stats-crosstab",
+      area: "action",
+    },
+    {
+      summary: "Click table rows to filter",
+      description: "Click on table rows to drill drown into those actions.",
+      hotspot: "stats-table",
+      area: "action",
+    },
+    {
+      summary: "Click and drag histogram to filter",
+      description:
+        "Clicking and dragging on a range will filter only matching the actions.",
```

Comment:
This sounds like 'the filter will only be applied to the highlighted range', is that right?

Reply:
bump

Reply:
Rewritten.